frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewMetaData/FeatureOverviewMetaData.tsx

```diff
@@ -156,6 +166,13 @@ const FeatureOverviewMetaData = () => {
                         <DependencyRow feature={feature} />
                     ) : null}
                     <TagRow feature={feature} />
+                    {onEnvironmentVisibilityChange ? (
+                        <EnvironmentVisibilityMenu
+                            environments={feature.environments || []}
+                            hiddenEnvironments={hiddenEnvironments || []}
+                            onChange={onEnvironmentVisibilityChange}
+                        />
+                    ) : null}
```

Comment:
Why is the onEnvironmentVisibilityChange function optional? Are there cases where we don't want you to do that?

Reply:
For tests that. We have some "parent flag" tests that don't pass environments.

Reply:
Hmm. I'm not sure I want the tests to determine whether something should be required or not. And in this case, seems like we fall back to an empty list of environments anyway? Optionally, you could keep the function optional but give it a default impl that does nothing?